routes/products.js

```diff
@@ -325,7 +325,7 @@ router.put('/:productId', auth, async (req, res) => {
 router.get('/:productId', auth, async (req, res) => {
   try {
     if (!verify.checkIfValidUUID(req.params.productId)) {
-      res.status(422).json({
+      res.status(404).json({
```

Comment:
@felkourdi This change is not required as this condition checks if the provided product id is a valid UUID or not. For cases where the product id is not found is handled at line 353